src/mono/mono/metadata/marshal-lightweight.c

```diff
@@ -2432,13 +2453,7 @@ emit_unsafe_accessor_ctor_wrapper (MonoMethodBuilder *mb, MonoMethod *accessor_m
 		return;
 	}
 
-	MonoGenericContext context = { NULL, NULL };
-	if (mono_class_is_ginst (target_class))
-		context.class_inst = mono_class_get_generic_class (target_class)->context.class_inst;
-	if (accessor_method->is_inflated)
-		context.method_inst = mono_method_get_context (accessor_method)->method_inst;
-	if ((context.class_inst != NULL) || (context.method_inst != NULL))
-		target_method = mono_class_inflate_generic_method_checked (target_method, &context, find_method_error);
+	target_method = inflate_method (target_class, target_method, accessor_method, find_method_error);
```

Comment:
we should handle `!is_ok(find_method_error)`.  emit a throwing method body or something.

Reply:
I see other places where `mono_class_inflate_generic_method_checked` usually assert if there is an error. I followed that fashion.